src/main/java/uk/gov/ida/verifyserviceprovider/dto/NonMatchingAttributes.java

```diff
@@ -70,20 +70,20 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) return false;
 
         NonMatchingAttributes that = (NonMatchingAttributes) o;
-        if (firstName != null ? !firstName.equals(that.firstName) : that.firstName != null) return false;
+        if (firstNames != null ? !firstNames.equals(that.firstNames) : that.firstNames != null) return false;
```

Comment:
All this could be simplified with:
```java
        return Objects.equals(firstNames, that.firstNames) &&
                Objects.equals(middleNames, that.middleNames) &&
                Objects.equals(surnames, that.surnames) &&
                Objects.equals(datesOfBirth, that.datesOfBirth) &&
                Objects.equals(gender, that.gender) &&
                Objects.equals(addresses, that.addresses);
```

Reply:
Yeah, that's true. This is auto-generated by Intellij so I'm going to leave it alone now.